Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py

```diff
@@ -1599,6 +1653,110 @@ def get_impacted_resources(client: PrismaCloudComputeClient, args: dict) -> Comm
         raw_response=raw_response
     )
 
+def get_waas_policies(client: PrismaCloudComputeClient, args: dict) -> CommandResults:
+    """
+    Get the WAAS policies.
+    Implement the command 'prisma-cloud-compute-get-waas-policies'
+
+    Args:
+        client (PrismaCloudComputeClient): prisma-cloud-compute client.
+        args (dict): prisma-cloud-compute-get-waas-policies command arguments
+
+    Returns:
+        CommandResults: command-results object.
+    """
+    policies = client.get_waas_policies()
+    entry = []
+    for rule in policies.get("rules") or {}:
+        for spec in rule.get("applicationsSpec") or {}:
+            formatted_waas_policy = {
+                "SQLInjection": spec.get("sqli").get("effect"),
+                "CrossSiteScriptingXSS": spec.get("xss").get("effect"),
+                "OSCommandInjetion": spec.get("cmdi").get("effect"),
+                "CodeInjection": spec.get("codeInjection").get("effect"),
+                "LocalFileInclusion": spec.get("lfi").get("effect"),
+                "AttackToolsAndVulnScanners": spec.get("attackTools").get("effect"),
+                "Shellshock": spec.get("shellshock").get("effect"),
+                "MalformedHTTPRequest": spec.get("malformedReq").get("effect"),
+                "ATP": spec.get("networkControls").get("advancedProtectionEffect"),
+                "DetectInformationLeakage": spec.get("intelGathering").get("infoLeakageEffect")
+            }
+            data = {
+                "Name": rule.get("name"),
+                "WaasPolicy": formatted_waas_policy
+            }
+
+            entry.append(CommandResults(
+                outputs_prefix="PrismaCloudCompute.Policies",
+                outputs_key_field="Name",
+                outputs=data,
+                readable_output=tableToMarkdown(data["Name"], data["WaasPolicy"]),
+                raw_response=policies
+            ))
+
+    return entry
+
+
+def update_waas_policies(client: PrismaCloudComputeClient, args: dict) -> CommandResults:
+    """
+    Update the WAAS policy.
+    Implement the command 'prisma-cloud-compute-update-waas-policies'
+
+    Args:
+        client (PrismaCloudComputeClient): prisma-cloud-compute client.
+        args (dict): prisma-cloud-compute-update-waas-policies command arguments
+
+    Returns:
+        CommandResults: command-results object.
+    """
+    waas_settings = ["sqli", "xss", "attackTools", "shellshock", "malformedReq", "cmdi", "lfi", "codeInjection"]
+
+    policy = args.get("policy")
+
+    for index, rule in enumerate(policy["rules"]):
```

Comment:
```suggestion
    for index, rule in enumerate(policy.get("rules")):
```

always use `.get` where possible to avoid `KeyError` exceptions.
that is true for all other places in the code :)